build.gradle

```diff
@@ -74,6 +75,7 @@ if (hasProperty('buildScan')) {
     buildScan {
         termsOfServiceUrl = 'https://gradle.com/terms-of-service'
         termsOfServiceAgree = 'yes'
+        publishAlways()
```

Comment:
I'm not sure, but I think this will help with the issue @ddietterich ran into where the build scan is not automatically generated.